src/icon-manifest/supportedExtensions.ts

```diff
@@ -378,6 +378,7 @@ export const extensions: IFileCollection = {
     { icon: 'nsi', extensions: [], languages: [languages.nsis], format: FileFormat.svg },
     { icon: 'nuget', extensions: ['nupkg', 'nuspec', 'psmdcp'], format: FileFormat.svg },
     { icon: 'nunjucks', extensions: ['nunj', 'njs'], languages: [languages.nunjucks], format: FileFormat.svg },
+    { icon: 'nuxt', extensions: [''], filenamesGlob: ['nuxt.config'], extensionsGlob: ['js', 'ts'], filename: true, format: FileFormat.svg },
```

Comment:
Do we need the empty string in `extensions`?

Reply:
Opps. No.